Firebase.Crashlytics/source/Firebase.Crashlytics/ApiDefinition.cs

```diff
@@ -0,0 +1,314 @@
+		[Export ("bundleVersion")]
+		string BundleVersion { get; }
+
+		// @property (readonly, copy, nonatomic) NSString * _Nonnull bundleShortVersionString;
+		[Export ("bundleShortVersionString")]
+		string BundleShortVersionString { get; }
+
+		// @property (readonly, copy, nonatomic) NSDate * _Nonnull dateCreated;
+		[Export ("dateCreated", ArgumentSemantic.Copy)]
+		NSDate DateCreated { get; }
+
+		// @property (readonly, copy, nonatomic) NSString * _Nonnull OSVersion;
+		[Export ("OSVersion")]
+		string OSVersion { get; }
+
+		// @property (readonly, copy, nonatomic) NSString * _Nonnull OSBuildVersion;
+		[Export ("OSBuildVersion")]
+		string OSBuildVersion { get; }
+
+		// @property (readonly, assign, nonatomic) BOOL isCrash;
+		[Export ("isCrash")]
+		bool IsCrash { get; }
+
+		// -(void)setObjectValue:(id _Nullable)value forKey:(NSString * _Nonnull)key;
+		[Export ("setObjectValue:forKey:")]
+		void SetObjectValue ([NullAllowed] NSObject value, string key);
+
+		// @property (copy, nonatomic) NSString * _Nullable userIdentifier;
+		[NullAllowed]
+		[Export ("userIdentifier")]
+		string UserIdentifier { get; set; }
+
+		// @property (copy, nonatomic) NSString * _Nullable userName;
+		[NullAllowed]
+		[Export ("userName")]
+		string UserName { get; set; }
+
+		// @property (copy, nonatomic) NSString * _Nullable userEmail;
+		[NullAllowed]
+		[Export ("userEmail")]
+		string UserEmail { get; set; }
+	}
+
+	// @interface CLSStackFrame : NSObject
+	[BaseType (typeof (NSObject), Name = "CLSStackFrame")]
+	interface StackFrame {
+		// +(instancetype _Nonnull)stackFrame;
+		[Static]
+		[Export ("stackFrame")]
+		StackFrame Create ();
+
+		// +(instancetype _Nonnull)stackFrameWithAddress:(NSUInteger)address;
+		[Static]
+		[Export ("stackFrameWithAddress:")]
+		StackFrame Create (nuint address);
+
+		// +(instancetype _Nonnull)stackFrameWithSymbol:(NSString * _Nonnull)symbol;
+		[Static]
+		[Export ("stackFrameWithSymbol:")]
+		StackFrame Create (string symbol);
+
+		// @property (copy, nonatomic) NSString * _Nullable symbol;
+		[NullAllowed, Export ("symbol")]
+		string Symbol { get; set; }
+
+		// @property (copy, nonatomic) NSString * _Nullable rawSymbol;
+		[NullAllowed, Export ("rawSymbol")]
+		string RawSymbol { get; set; }
+
+		// @property (copy, nonatomic) NSString * _Nullable library;
+		[NullAllowed, Export ("library")]
+		string Library { get; set; }
+
+		// @property (copy, nonatomic) NSString * _Nullable fileName;
+		[NullAllowed, Export ("fileName")]
+		string FileName { get; set; }
+
+		// @property (assign, nonatomic) uint32_t lineNumber;
+		[Export ("lineNumber")]
+		uint LineNumber { get; set; }
+
+		// @property (assign, nonatomic) uint64_t offset;
+		[Export ("offset")]
+		ulong Offset { get; set; }
+
+		// @property (assign, nonatomic) uint64_t address;
+		[Export ("address")]
+		ulong Address { get; set; }
+	}
+
+	// @interface Crashlytics : NSObject
+	[BaseType (typeof (NSObject))]
+	interface Crashlytics {
```

Comment:
Same, it seems you need to provide n Api key, is `init` valid here? Maybe it is missing`[DisableDefaultCtor]`